### PR TITLE
CEDS-1608 - add legal declaration questions to the declaration submit…

### PR DESCRIPTION
**Pull request by SteveSugden**
… page

### CODE REVIEW

File: app/utils/validators/forms/FieldValidator.scala [hunk 1]
```diff
@@ -53,6 +53,8 @@ object FieldValidator {
 
   val nonEmpty: String => Boolean = (input: String) => input.trim.nonEmpty
 
+  val isTrue: Boolean => Boolean = (input: Boolean) => input.equals(true)
```
Comment (PatrykRudnicki) [inline app/utils/validators/forms/FieldValidator.scala]: Hmm, `input.equals(true)` => if true return true, if false return false, so can you just return `input`? There is no point to check if it's equal to `true`

File: test/forms/declaration/LegalDeclarationSpec.scala [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ * limitations under the License.
+ */
+
+package forms.declaration
+
+import helpers.views.declaration.WarehouseIdentificationMessages
+import play.api.libs.json._
+import unit.base.UnitSpec
+
+class LegalDeclarationSpec extends UnitSpec with WarehouseIdentificationMessages {
```
Comment (PatrykRudnicki) [inline test/forms/declaration/LegalDeclarationSpec.scala]: Why are you extending `WarehouseIdentificationMessages`?

**Comment by PatrykRudnicki** [comment]
Looks good, will approve it after answers :)

File: app/utils/validators/forms/FieldValidator.scala (hunk 1 above)
Comment (SteveSugden) [reply app/utils/validators/forms/FieldValidator.scala]: That's true.  Readability?  :-)

File: test/forms/declaration/LegalDeclarationSpec.scala (hunk 2 above)
Comment (SteveSugden) [reply test/forms/declaration/LegalDeclarationSpec.scala]: Because I forgot to delete from the Spec I copied to create this one.

**Pull request merged**